shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-common/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java

```diff
@@ -167,7 +167,23 @@ public String getCipherColumn(final String logicTable, final String logicColumn)
      * @return cipher column or not
      */
     public boolean isCipherColumn(final String tableName, final String columnName) {
-        return tables.containsKey(tableName) && tables.get(tableName).getCipherColumns().contains(columnName);
+        return tables.containsKey(tableName) && isCipherColumn(tables.get(tableName).getCipherColumns(), columnName);
+    }
+
+    /**
+     * Is cipher column or not.
+     *
+     * @param cipherColumns cipher columns
+     * @param columnName column name
+     * @return cipher column or not
+     */
+    private boolean isCipherColumn(final Collection<String> cipherColumns, final String columnName) {
+        for (String each : cipherColumns) {
```

Comment:
It is better to use java8 lambda instead of for each

Reply:
ok, I will make a change.